lib/transforms/workflow.js

```diff
@@ -52,19 +66,14 @@ module.exports = (payload) => {
         state: mapStatus(workflow_run),
         lastUpdated: workflow_run.updated_at,
         issueKeys,
-        references: workflow_run.pull_requests.map(pr => (
```

Comment:
`map` throws an error is `pull_requests` is empty. Also, `references` can't be an empty array, so moved this check to the bottom.